layouts/shortcodes/page/header.html

```diff
@@ -1,13 +1,13 @@
 {{ $title := $.Page.Params.title -}}
 {{ $desc  := $.Page.Params.description | markdownify -}}
 <div class="text-left">
-  <h1 class="display-1 mb-5">
```

Comment:
Removed mb-5 (margin-bottom) from h1 to avoid unnecessary spacing when h3 is absent. Applied mt-5 (margin-top) to h3 instead to ensure consistent spacing between h1 and h3 only when h3 is present.
eg: On community page h3 is absent after h1, hence the spacing should not be applied.